x-pack/plugins/apm/server/plugin.ts

```diff
@@ -132,8 +134,18 @@ export class APMPlugin
               [PROCESSOR_EVENT]: {
                 type: 'keyword',
               },
+              [AGENT_NAME]: {
+                type: 'keyword',
+              },
+              [SERVICE_LANGUAGE_NAME]: {
+                type: 'keyword',
+              },
+              labels: {
```

Comment:
Should we include this is `elasticsearch_fieldnames.ts`?
I see we have there `LABEL_NAME = 'labels.name'`

Reply:
Yes, @MiriamAparicio, will you do that?

Reply:
@sqren but this is just `labels`, not `labels.name`, or am I missing something?